src/lib/core/drawer/state-listener.component.ts

```diff
@@ -17,11 +55,12 @@ export class StateListener implements OnInit, OnDestroy {
     }
 
     public isOpen(): boolean {
-        return this.drawerService.isDrawerOpen();
+        // console.log(this.drawerService.getNumberOfDrawerInstances());
```

Comment:
Please review console.log

Reply:
Changes looks good only 1 small comment.

Reply:
I've removed that extra console log, good catch.